pkg/cmd/cli/cmd/deploy.go

```diff
@@ -228,6 +229,7 @@ func (o DeployOptions) deploy(config *deployapi.DeploymentConfig, out io.Writer)
 	_, err = o.osClient.DeploymentConfigs(config.Namespace).Update(config)
 	if err == nil {
 		fmt.Fprintf(out, "Started deployment #%d\n", config.Status.LatestVersion)
+		fmt.Fprintf(out, "Use '%s logs -f dc/%s' to track its progress.\n", o.baseCommandName, config.Name)
```

Comment:
Does this really need to be on a new line?  Does it look better or worse?  If you put it on one line, can you shorten the second sentence?

If it's so common that someone wants to follow their deployment with the logs, why wouldn't we include that in deploy or do it by default?  What percentage of users will want to view the logs of the deployment?